Userland/Libraries/LibWeb/Infra/ByteSequences.cpp

```diff
@@ -25,4 +25,35 @@ void byte_uppercase(ByteBuffer& bytes)
         bytes[i] = to_ascii_uppercase(bytes[i]);
 }
 
+// https://infra.spec.whatwg.org/#byte-sequence-prefix
+// https://infra.spec.whatwg.org/#byte-sequence-starts-with
+bool is_prefix_of(ByteBuffer const& potential_prefix, ByteBuffer const& input)
```

Comment:
not sure about this change—reading it is not really obvious. maybe sticking to `starts_with(a, b)` would be better? (now youʼre inverting it making it more brain straining)